openghg/util/_util.py

```diff
@@ -235,3 +236,20 @@ def multiple_inlets(site: str) -> bool:
             return True
 
     return len(heights) > 1
+
+
+def sort_by_filenames(filepath: Union[list[Path], str]) -> Union[list[Path], str]:
+    """
+    Sorting time on filename basis
+
+    Args:
+        filepath: Path to the fil
+
+    Returns:
+        list[Path]: List of sorted paths
+    """
+
+    filepath = sorted([Path(f) for f in filepath])
```

Comment:
This doesn't account for a `str` being passed at the moment (would loop over the characters in a `str`) - I think the if statement you had in a previous version did though? You may also want to look at using `multiPathType` to keep this consistent.